src/pages/Home.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import {
+  withPageWrapper,
+  InvoicesTable,
+  IInvoicesTableRender,
+} from "../components";
+
+const InvoicesTableWithPageWrapper = withPageWrapper(InvoicesTable);
+
+export default function Home() {
+  const [posts, setPosts] = useState([]);
```

Comment:
you don't use full typescript support here - you should type the component type (functional) and type state values (to say what values we can store in this state)